### ISSUE TITLE
"AllCops Include" is removing all default files

### ISSUE THREAD

**Issue by fabioperrella** [rubocop-hq/rubocop#6021]
## Expected behavior

Using **Include** inside the **AllCops** should not overwrite the default inclusion rules.

I've just updated the version from 0.52.1 to 0.57.2 and that behavior has changed.

## Actual behavior

Using **Include** inside the **AllCops** is overwriting the default inclusion rules

## Steps to reproduce the problem

```bash
$ touch lala.rb popo.rb other
```
- create a .rubocop.yml with this content
```yaml
AllCops:
  Include:
    - 'other'
```
- run `be rubocop -L`

I expect to list all the 3 files, but it is listing only the `other`

## RuboCop version

0.57.2 (using Parser 2.5.1.0, running on ruby 2.3.3 x86_64-linux)

**Comment by yedennek** [comment]
We are experiencing this same issue with the following Rubocop version:

0.57.1 (using Parser 2.5.1.0, running on ruby 2.5.0 x86_64-darwin15)

**Comment by enthusiasmus** [comment]
same here with 0.57.2

**Comment by bbatsov** [comment]
That's intentional - see https://github.com/rubocop-hq/rubocop/pull/5882and  https://github.com/rubocop-hq/rubocop/issues/5917

**Comment by fabioperrella** [comment]
@bbatsov I had to add the code below to my rubocop.yml to be compatible with the previous version:

```yml
inherit_mode:
  merge:
    - Exclude
    - Include
```

What do you think to add this hint in some doc to help people upgrading from older versions?

**Comment by bbatsov** [comment]
I thought @jonas054 updated the docs with respect to this. If that's not the case - we should certainly do it. 

I know this change was a bit painful, but it had to be done, as it simplified a lot the internal of RuboCop and removed the non-transparent defaults that were hardcoded in the past.

**Comment by fabioperrella** [comment]
@bbatsov  I think [this doc](https://github.com/rubocop-hq/rubocop/blob/master/manual/configuration.md#merging-arrays-using-inherit_mode) is clear about the usage of `inherit_mode`, but maybe there is not a instruction for people upgrading rubocop from older versions. 

IMO this change has break the compatibility with the last version and it deserves a warning about it.

What do you think to add it in CHANGELOG?

**Comment by bbatsov** [comment]
Yeah, I'm fine with this. Some big "BREAKING" warning a link to the solution.
